packages/components/src/ItemList.test.tsx

```diff
@@ -248,35 +220,56 @@ describe('mouse', () => {
       onSelectionChange.mockClear();
       onSelect.mockClear();
 
-      rightClickItem(itemList, 5);
+      rightClickItem(5);
 
       expect(onSelect).not.toHaveBeenCalled();
       expect(onSelectionChange).not.toHaveBeenCalled();
-
-      itemList.unmount();
     });
   });
 });
 
+function checkFocus(elementList, index) {
+  for (let i = 0; i < 20; i += 1) {
+    if (i === index) {
+      expect(elementList[i]).toHaveFocus();
+    } else {
+      expect(elementList[i]).not.toHaveFocus();
+    }
+  }
+}
 it('handles keyboard up and down properly', () => {
   const itemList = makeItemList();
 
-  expect(itemList.state('focusIndex')).toBe(null);
+  const correctList = itemList.baseElement.firstChild.firstChild;
```

Comment:
You're getting the `item-list-scroll-pane` here, instead of getting it this way restructure it a bit to get by class name instead (we'd prefer test id or role but the react-window library we're using doesn't appear to support that):
```
  const { container } = makeItemList();
  const scrollPanes = container.getElementsByClassName('item-list-scroll-pane');
  expect(scrollPanes.length).toBe(1);
  const list = scrollPanes[0];
  const items = screen.getAllByRole('presentation').splice(1);
```
Then all the fireEvent.keyDowns can be shortened back down to one line.